KomaMRIFiles/test/runtests.jl

```diff
@@ -35,6 +35,8 @@ using TestItems, TestItemRunner, KomaMRIBase
         @test shape == shape2
 
         # Test label capability
+        using KomaMRIBase
```

Comment:
I don't know why I have to add that here. KomaMRIBase is already imported no ?

Reply:
Is `get_label` not defined if you don't? Maybe you forgot to export some function in KomaMRIBase.